backend/src/app.ts

```diff
@@ -319,7 +319,48 @@ app.post('/api/add-like', authenticate, likeHandler(false));
 
 app.post('/api/remove-like', authenticate, likeHandler(true));
 
-app.put('/api/update-review-status/:reviewDocId/:newStatus', async (req, res) => {
+// Endpoint to update a review by its document ID
+app.put('/api/edit-review/:reviewDocId', async (req, res) => {
```

Comment:
Generally the convention is you get the id and then you do the action, so I think `:reviewDocId/edit` would be more apt.

Same with line 348